tobac/tests/test_import.py

```diff
@@ -4,3 +4,10 @@
 
 def test_dummy_function():
     assert 1 == 1
+
+
+def test_version():
+    """Test to make sure that we have a version number included.
+    If it's not, this should result in an error.
+    """
+    assert tobac.__version__ == tobac.__version__
```

Comment:
Hm, good point. I don't know if test functions automatically `return` though? I will test that.

Reply:
Returning the version also works, but it does not seem so common to return values in tests - maybe because the only case in which the test fails is when the returned value is not defined? 

Alternatively `assert type(tobac.__version__) is str` ? That would also ensure the version attribute is the right type and not mistakenly `1.4` as a float or something. 

@snilsn Good point with the exception. I think so far we have only used `assert` statements that result in `AssertationError` when the tests fail, which is maybe good enough if the test function is properly described? Do you use  `pytest.raises(Exception)` also for that or do you use it only when you actually test if an exception is raised properly?

Reply:
After a bit of research I think the minimal be the way to do is just 

`version_number = tobac.__version__`

without a return. Or if we wanna be a little bit more elaborate and stick with  `assert`:

```
    try:
        version_number = tobac.__version__
    except Exception:
        assert False, 'no version number'
```

But we could also go with Julia's idea.

@JuliaKukulies Yes, I think `pytest.raises(Exception)` is mainly for testing if expected exceptions are raised correctly, but I'm really not an expert for testing. There could be clever ways to use this. Maybe we could make our tests more usefull by adding messages to our asserts, especially  if there is more than one within a function.

Reply:
OK, I've now updated this code based on this discussion. Thanks much for the thoughts, @snilsn @JuliaKukulies

Reply:
Thanks also for clarifying `pytest.raises(Exception)` !